### PR TITLE
Clean up spelling of nonexistent

### PR DESCRIPTION
**Pull request by othercorey**
Simple cleanup of the spelling in tests and comments.

### CODE REVIEW

File: src/View/Form/EntityContext.php [hunk 1]
```diff
@@ -629,7 +629,7 @@ protected function _getValidator(array $parts): Validator
      *
      * @param string[]|string|\Cake\Datasource\EntityInterface $parts Each one of the parts in a path for a field name
      * @param bool $fallback Whether or not to fallback to the last found table
-     *  when a non-existent field/property is being encountered.
+     *  when a nonexistent field/property is being encountered.
```
Comment (dereuromark) [inline src/View/Form/EntityContext.php:632]: Interesting, that my spelling recommendation given is the former
https://www.dict.cc/deutsch-englisch/nicht+existent.html

Comment (othercorey) [reply src/View/Form/EntityContext.php:632]: Some dictionaries have it, but I was mainly cleaning up the spelling of existent and spacing. So, just decided to make them all consistently use the full word.

Comment (dereuromark) [reply src/View/Form/EntityContext.php:632]: Given https://english.stackexchange.com/questions/237113/nonexistent-non-existent-or-non-existent I would tend to agree with you :)
The prosecution rests your honor :P

File: tests/TestCase/View/Helper/FormHelperTest.php [hunk 2]
```diff
@@ -7533,52 +7533,52 @@ public function testControlWithFractional(string $type)
     }
 
     /**
-     * testForMagicControlNonExistingNorValidated method
+     * testForMagicControlNonExistentNotValidated method
      *
      * @return void
      */
-    public function testForMagicControlNonExistingNorValidated()
+    public function testForMagicControlNonExistentNotValidated()
```
Comment (garas) [inline tests/TestCase/View/Helper/FormHelperTest.php:7540]:
```suggestion
    public function testForMagicControlNonexistentNotValidated()
```

File: tests/TestCase/View/Helper/FormHelperTest.php [hunk 3]
```diff
@@ -7533,52 +7533,52 @@ public function testControlWithFractional(string $type)
     }
 
     /**
-     * testForMagicControlNonExistingNorValidated method
+     * testForMagicControlNonExistentNotValidated method
```
Comment (garas) [inline tests/TestCase/View/Helper/FormHelperTest.php:7536]:
```suggestion
     * testForMagicControlNonexistentNotValidated method
```

File: tests/TestCase/View/Helper/FormHelperTest.php (hunk 2 above)
Comment (othercorey) [reply tests/TestCase/View/Helper/FormHelperTest.php:7540]: I replaced those specifically for you.

**Pull request merged**